workflows/variables.tf

```diff
@@ -1,14 +1,20 @@
 variable "DEPLOY_NAME" {
-  type = string
+  type    = string
   default = "daac"
 }
 
 variable "MATURITY" {
-  type = string
+  type    = string
   default = "dev"
 }
 
 variable "DIST_DIR" {
-  type = string
+  type    = string
   default = "../dist"
 }
+
+variable "cloudwatch_log_retention_periods" {
```

Comment:
I could be missing something - but this is unused in the `workflows` module, right?  I don't see it referenced in `tf-modules.workflow` in Cumulus.

Reply:
Good catch.  I'll remove it.